.gitlab-ci.yml

```diff
@@ -0,0 +1,71 @@
+---
+stages:
+  - build-containers
+  - tag
+  - deploy
+include:
+  - local: '/.gitlab-ci/build-image.yaml'
+  - local: '/.gitlab-ci/deploy.yaml'
+
+variables:
+  FAILFASTCI_NAMESPACE: connylegal
+  OCI_REPO: img.conny.dev
+  CONTAINER: conny/printer
+  IMAGE_NAME: $OCI_REPO/$CONTAINER
+  APPNAME: printer
+  APP: printer
+  DEFAULT_TAG: v$CI_COMMIT_REF_SLUG
+
+
+# BUILD IMAGE
+printer image:
+  extends: .build-container
+
+
+# TAGS IMAGES
+printer tag gitsha:
+  extends: .tag-container
+  variables:
+    DEST_TAG: $CI_COMMIT_SHORT_SHA
+  rules:
+    - if: '$CI_COMMIT_BRANCH == "gitlab_ci"'
```

Comment:
I believe this was intentional for testing purposes, but for merging it would `master`  right?